icu4c/source/common/cmemory.h

```diff
@@ -800,7 +844,8 @@ class MaybeStackVector : protected MemoryPool<T, stackCapacity> {
     }
 
     /**
-     * Append all the items from another MaybeStackVector to this one.
+     * Append all the items from another MaybeStackVector to this one via
+     * copy-construction.
```

Comment:
I'm not really sure what it is that you want to convey with your edit to this comment. Is maybe "Append copies of all items from another MaybeStackVector to this one." what you want to say?

Reply:
Yes, that sounds much better, thanks!